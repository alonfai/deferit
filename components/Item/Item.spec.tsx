// import { render, fireEvent } from '@testing-library/react-native';
// import ItemSummary from './ItemSummary';

describe('<Item />', () => {
  it('demo', () => {
    expect(1).toEqual(1);
  });
});
