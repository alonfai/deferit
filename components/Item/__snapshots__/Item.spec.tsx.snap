// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Item /> Render successfully 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "borderRadius": 10,
      "borderWidth": 0.5,
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-around",
      "marginHorizontal": 16,
      "marginVertical": 8,
      "padding": 8,
    }
  }
>
  <View
    style={
      Object {
        "paddingHorizontal": 10,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
    >
      <Image
        source={
          Object {
            "uri": "http://www.image-thumbnail.jpg",
          }
        }
        style={
          Object {
            "height": 100,
            "width": 100,
          }
        }
      />
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "column",
        "marginHorizontal": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "bold",
          "textDecorationLine": "underline",
        }
      }
    >
      Status
    </Text>
    <View
      accessibilityLabel="Status Label"
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <Text />
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "flexDirection": "column",
        "marginHorizontal": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "bold",
          "textDecorationLine": "underline",
        }
      }
    >
      Extra
    </Text>
    <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
      >
        <Text />
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "column",
        "marginHorizontal": 10,
        "width": 130,
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "bold",
          "textDecorationLine": "underline",
        }
      }
    >
      Summary
    </Text>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "flexDirection": "row",
          "flexWrap": "wrap",
          "marginVertical": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
        amount
      </Text>
      <Text
        style={
          Object {
            "paddingLeft": 5,
          }
        }
      >
        $10
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "flexDirection": "row",
          "flexWrap": "wrap",
          "marginVertical": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
        status
      </Text>
      <Text
        style={
          Object {
            "paddingLeft": 5,
          }
        }
      >
        Processing
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "flexDirection": "row",
          "flexWrap": "wrap",
          "marginVertical": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
        date
      </Text>
      <Text
        style={
          Object {
            "paddingLeft": 5,
          }
        }
      >
        01/01/2022
      </Text>
    </View>
  </View>
</View>
`;
